pyccl/core.py

```diff
@@ -1023,3 +1023,32 @@ def _set_linear_power_from_arrays(self, a_array=None, k_array=None,
             self.a_array = a_array
             self.k_array = k_array
             self.pk_array = pk_array
+
+
+class CosmologyPlanck18LCDM(Cosmology):
+    """A cosmology roughly corresponding to the best-fit parameters from
+    the 2018 Planck release (arXiv:1807.06209).
+
+    Args:
+        **kwargs (dict): a dictionary of parameters passed as arguments
+            to the `Cosmology` constructor. It should not contain any of
+            the LambdaCDM parameters (`"Omega_c"`, `"Omega_b"`, `"n_s"`,
+            `"sigma8"`, `"A_s"`, `"h"`), since these are fixed.
+    """
+    def __init__(self, **kwargs):
```

Comment:
This use of `**kwargs` and the docstring is great for developers since it is very easy code to write, but horrible for users since now they cannot find any of the information they might need about how to use this object.

Reply:
I thought this was relatively standard (see e.g. https://matplotlib.org/devdocs/api/_as_gen/matplotlib.pyplot.imshow.html). Note that the documentation should show a link to the default cosmology object, where the full set of parameters is listed.

Reply:
Yeah the matplotlib docs are awful to use.

Reply:
It feels silly to repeat the full documentation and set of parameters here, right?

Reply:
shrug - up to you

You can also access the __doc__ magic of the other class and paste it in.